### ISSUE TITLE
uib-rating: The WAI-ARIA property aria-valuetext is not supported on elements with WAI-ARIA role button

### ISSUE THREAD

**Issue by tomkdgun** [angular-ui/bootstrap#5571]
The issues forum is __NOT__ for support requests. It is for bugs and feature requests only.
Please read https://github.com/angular-ui/bootstrap/blob/master/CONTRIBUTING.md and search
existing issues (both open and closed) prior to opening any new issue and ensure you follow the instructions therein.

### Bug description:
The WAI-ARIA property aria-valuetext is not supported on elements with WAI-ARIA role button.
This is for uib-rating star element, and is related with accessibility. 
https://www.w3.org/TR/wai-aria/states_and_properties#aria-valuetext

<i class="glyphicon ng-scope glyphicon-star" **aria-valuetext**="one" ng-attr-title="{{r.title}}" ng-class="$index < value && (r.stateOn || 'glyphicon-star') || (r.stateOff || 'glyphicon-star-empty')" ng-click="rate($index + 1)" ng-mouseenter="enter($index + 1)" ng-repeat-end="" role="**button**" tabindex="0" title="one"></i>

### Version of Angular, UIBS, and Bootstrap

Angular: 1.4.9

UIBS: 1.2.1

Bootstrap: 3.3.6

**Comment by wesleycho** [comment]
Looks like `aria-valuetext` was on the wrong element - it was on each star element, but it should have been on the top level element in the template.
